.github/workflows/release.yaml

```diff
@@ -22,14 +22,17 @@ jobs:
       IMAGE_NAMESPACE: ${{ github.repository }}
       VERSION: ${{ github.event.inputs.release }}
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
         fetch-depth: 0  # Fetch all history, needed for release note generation.
     # Install tools
-    - uses: actions/setup-go@v2
+    - name: Install Go
+      uses: actions/setup-go@v3
       with:
-        go-version: 1.18.x
-    - uses: sigstore/cosign-installer@v2.3.0
+        go-version: ${{ matrix.go-version }}
```

Comment:
seems not needed, could you keep it at 1.18, or maybe move it to 1.19 together with https://github.com/shipwright-io/operator/issues/104

Reply:
Done.